modules/gridmap/editor/grid_map_editor_plugin.cpp

```diff
@@ -1199,6 +1199,8 @@ void GridMapEditor::_notification(int p_what) {
 					RS::get_singleton()->instance_set_transform(grid_instance[i], xf * edit_grid_xform);
 				}
 				grid_xform = xf;
+				_update_cursor_transform();
+				_update_selection_transform();
```

Comment:
Looks simple enough, but as a general question, without having looked in depth at the code or tested it: I assume this means that dragging the GridMap to change its position will cause updates of the cursor and selection on every single tick. Is the performance cost here reasonable, or is this something we'd want to defer to avoid updating too frequently?

Reply:
From a look at the code, these calls do not seem to do much. I put them inside a for loop in an unoptimized build (dev_build=yes, dev_mode=yes and checked the disassembly in the debugger to ensure the loop wasn't removed) and had to use 100000 iterations for it to make a noticeable difference on my 5yo PC, so i'd *guess* the performance is fine.